samples/grpc/local-drone-control-scala/src/main/scala/local/logback/NativeImageAsyncAppender.scala

```diff
@@ -0,0 +1,33 @@
+package local.logback
+
+import ch.qos.logback.classic.AsyncAppender
+import ch.qos.logback.classic.spi.ILoggingEvent
+
+/**
+ * Lazy version of AsyncAppender for native images, as threads can't be started during native image build.
+ */
+class NativeImageAsyncAppender extends AsyncAppender {
```

Comment:
Needed because logging has to be configured at build time. Could also not worry about async logging in native images, just use the stdout console appender.